manifest/manifest.mjs

```diff
@@ -5,11 +5,14 @@ import * as utils from "../utilities/shared.mjs"
 export async function findTheManifestByID(id=null){
    let manifest = null
    if(!utils.validateProjectID(id)) return manifest
```

Comment:
Why is `manifest` in here at all? You set to null and then return null.
You probably want to return the validation error (which looks like `false` if I'm reading it right).

Reply:
No.  The function receives a project ID as input and returns null or the matching manifest.  It mocks some Manifest logic.  Given a project ID, go get the Manifest.
```
const manifestObj = await logic.findTheManifestByID(id)
```
Is either null or a Manifest